crates/bevy_render/src/camera/camera.rs

```diff
@@ -436,13 +465,15 @@ impl NormalizedRenderTarget {
             NormalizedRenderTarget::Image(image_handle) => {
                 images.get(image_handle).map(|image| image.texture_format)
             }
+            NormalizedRenderTarget::TextureView(_) => Some(TextureFormat::bevy_default()),
```

Comment:
I don't think this line is correct. As far as I know, it's possible for a TextureView to be a different format.

Reply:
Ok we should check that, my understanding was that the `TextureView` is opaque and we dont have access to the format, @kcking do you have thoughts on this?

Reply:
if it is something we want to track, maybe instead of storing a `(TextureView,UVec2)` tuple, we can store some struct

```rs
pub struct ManualTextureView {
    texture_view: TextureView,
    size: UVec2,
    format: TextureFormat,
}
```

Reply:
Agreed, storing the correct `TextureFormat` makes the most sense to me.